app/views/answer_mailer/notify_answer_likers_author.html.erb

```diff
@@ -0,0 +1,19 @@
+Hi! There's been a new answer posted for the question '<a href='https://publiclab.org<%= @answer.node.path(:question) %>'><%= @answer.node.title %></a>' that you <% if @answer.node.liked_by(@user.uid) %>liked<% else %>also answered<% end %>. Do NOT reply to this email; click this link to see the answer:
+
+<p>https://publiclab.org<%= @answer.node.path(:question) %>#a<%= @answer.id %></p>
+
+<p><b>Here is the answer that<a href="https://publiclab.org/profile/<%= @answer.author.name %>"><%= @answer.author.name %></a> wrote:</b></p>
+
+<hr /> 
+
+<p><%= raw auto_link(RDiscount.new(@answer.body_email).to_html) %></p>
+
+<hr />
+
+<div style="color:#aaa;">
+
+<p>You can comment on the answer at: https://publiclab.org<%= @answer.node.path(:question) %>#answer-<%= @answer.id %>-comment</p>
+
+<p>Report abuse to: web@publiclab.org</p>
```

Comment:
This will have to change to moderators@publiclab.org throughout, sorry -- recent change!